.snyk

```diff
@@ -9,5 +9,9 @@ ignore:
     - '*':
         reason: Believed we don't used the open SSL libraries. Will revisit
         expires: 2022-03-04T00:00:00.000Z
+  SNYK-ORACLE7-EXPAT-2427672:
+    - '*':
+        reason: Hoping this will be patched soon
```

Comment:
Can I request we re-word this, I feel like 'hope' doesn't read well as a basis for security decisions 😅 

```suggestion
        reason: Does not appear to be exploitable, accept risk in short term pending patch
```